frontend/class-json-ld.php

```diff
@@ -0,0 +1,270 @@
+<?php
+/**
+ * @package WPSEO\Frontend
+ */
+
+/**
+ * Class WPSEO_JSON_LD
+ *
+ * Outputs schema code specific for Google's JSON LD stuff
+ *
+ * @since 1.8
+ */
+class WPSEO_JSON_LD {
+
+	/**
+	 * @var array Holds the plugins options.
+	 */
+	public $options = array();
```

Comment:
I would make this private, because it look likes it isn't used outside this the scope of this class.